### ISSUE TITLE
Self test fails.

### ISSUE THREAD

**Issue by jkbonfield** [AlexanderVi/ont2cram#13]
I'm using Ubuntu 18.04.2 LTS (bionic) which comes with python 3.6.8.

However a fresh git clone of ont2cram fails its tests:

```
======================================================================
ERROR: test_round_trip_5__home_ubuntu_ont2cram_test_data_multi_read_0 (test.Ont2CramTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/parameterized/parameterized.py", line 392, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/home/ubuntu/ont2cram/test.py", line 110, in test_round_trip
    cram2ont.run( cram_path, self.fast5_restored_dir )
  File "/home/ubuntu/ont2cram/cram2ont.py", line 164, in run
    cram_to_fast5( input_file, output_dir )
  File "/home/ubuntu/ont2cram/cram2ont.py", line 137, in cram_to_fast5
    dtype=[(col_name, a.type)]
TypeError: a bytes-like object is required, not 'str'
```

Is this a known bug, or some incompatibility with my versions of things?
Any advice welcome.  Note I'm not a python programmer.

Edit: deleted second half.  Dumb user error. :-)

**Comment by jkbonfield** [comment]
That second error message appears to simply be because the test_data directory contains a variety of files and it wants everything to have identical keys, so is probably "user error".

However maybe the test data directory should have sub-directories split up per type of data so dumb users like myself don't get suckered into doing the wrong thing when testing.

I'm pleased to say it worked on some real data anyway.  :-)

**Comment by AlexanderVi** [comment]
This has been fixed in de25e3d ( line 347 ). Just checked on Ubuntu 18.04.3 LTS (python 3.6.9) - all tests pass.
